samples/matter/weather_station/README.rst

```diff
@@ -48,26 +48,30 @@ You can start testing after :ref:`building and running the sample <matter_weathe
 Remote testing in a network
 ===========================
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_remote_testing_start
-    :end-before: matter_door_lock_sample_remote_testing_end
+By default, the Matter accessory device has Thread disabled, and it must be paired with the Matter controller over Bluetooth LE to get configuration from it if you want to use the device within a Thread network.
+To do this, the device must be made discoverable over the Bluetooth LE (that is enabled automatically) and the controller must get the commissioning information from the Matter accessory device and provision the device into the network.
```

Comment:
"must be made" suggests that someone must do it. It would rather write sth like:
"The Bluetooth LE advertising starts automatically upon the device startup. The controller must get the ..."